docs/organizer-guide/vm-organize-quorum-meetup.md

```diff
@@ -0,0 +1,76 @@
+- **Did someone in the WTD community creatively solve a problem or develop a novel approach to a documentation task?** Sometimes people in the Write the Docs community have really interesting approaches to solving documentation problems. Sharing those problems and cool solutions can help others in their own sticky situations. Feel free to reach out to members of our community who might be interested in providing more details.
+- **Does someone who works for a company that makes documentation tools want to speak to our audience?** While we don't want every presentation to turn into a product demo, members of our community might be genuinely interested in hearing about new products or tools that are being developed to help solve documentation problems. Some product demos can be quite interesting and these kind of speakers will (of course) jump at the opportunity to speak to our community.
+
+If you're stuck, ping into the #meetup-organizers-quorum channel on Slack to brainstorm possible speakers. 
+
+In the future, we hope to provide a speaker database to help assist with this part of the process. If you'd like to volunteer to help organize this database, we're happy to help you get started. 
+
+## Confirm details
+
+When you have a speaker lined up and confirmed, you are ready to check with Quorum meeting volunteer leaders to discuss the speaker and topic and determine whether it's a good fit for a Quorum meetup event, and finalize details, including:
+
+*  Date and time for the presentation
+*  Presentation title and description
+*  Obtaining a brief speaker bio
+*  Any other concerns, materials, or information
+
+## Create event
+
+After the details have been nailed down, coordinate with Quorum volunteers organizers to schedule the Zoom event and Meetup event.
+
+## Publicize event
+
+After the Zoom and Meetup events are created, announce it to the rest of the Quorum meetup group, and remind Quorum members to join the meetup and help publicize it to their individual meetup groups members.
+
+Great ways to publicize the Quorum meetups in your local meetup groups include:
+
+*  Adding a description and meetup event link to Slack in your local meetup channel.
+*  Sending out an email via Meetup.org to your group with the meetup event info.
+*  Using Twitter or other social media avenues to further promote the meetup event.
+
+## Practice emcee-ing
+
+One week before your meetup, the meetup organizer hosts a dry-run of the event emceeing and general flow to work through the logistics and decide which ice-breaker questions you want to include---it only takes about 30 minutes! This dry run helps give you confidence in helming the big event and gives experienced Quorum volunteer leaders the information to effectively manage the event alongside you.
+
+You can also invite your speaker or speakers to the dry-run if you feel it would be advantageous. It can help you and your speakers feel at ease and it gives everyone a chance to ask questions.
```

Comment:
```suggestion
You can also invite your speaker or speakers to the technical check if you feel it would be advantageous. It can help you and your speakers feel at ease and it gives everyone a chance to ask questions.
```